apps/crl2p7.c
=======================================================
lhs: 100644 | 73e62702ce7e8ed797900c8148ac53bfcac891bf
rhs: 100644 | 1631258793ec96ae5d653be3b6ebb7bfa7f3c26d
---@@ -143,8 +143,6 @@ int crl2pkcs7_main(int argc, char **argv)
             }
         }
 
-    sk_OPENSSL_STRING_free(certflst);
-
     out = bio_open_default(outfile, 'w', outformat);
     if (out == NULL)
         goto end;
@@ -160,6 +158,7 @@ int crl2pkcs7_main(int argc, char **argv)
     }
     ret = 0;
  end:
+    sk_OPENSSL_STRING_free(certflst);
     BIO_free(in);
     BIO_free_all(out);
     PKCS7_free(p7);

---